scripts/check_compliance.py

```diff
@@ -588,6 +588,118 @@ def run(self):
             self.add_failure(msg.decode("utf-8"))
 
 
+class PyLint(ComplianceTest):
```

Comment:
How about `class PylintRequirements( )` to make clearer that the vast majority of Pylint checks are neither required nor even run?

Reply:
I wouldn't have any idea what `PylintRequirements` means personally. It runs `pylint`. :)

Reply:
Is there some way to add a GitHub message without also failing the build btw?

If we add some thing to give the user the complete `pylint` output later, then maybe it could be implemented in the same class.

Thinking something like

```python
self.message("Here is the complete pylint output. Check for anything that "
             "should be fixed.\n\n" + full_pylint_output)
```